src/DynamoCoreWpf/Views/Menu/PreferencesView.xaml.cs

```diff
@@ -374,6 +308,37 @@ private void ReloadCPython_Click(object sender, RoutedEventArgs e)
             dynViewModel.Model.OnRequestPythonReset(PythonServices.PythonEngineManager.CPython3EngineName);
         }
 
+        /// <summary>
+        /// This event is generated every time the user clicks a Radio Button in the Geometry Scaling section
+        /// The method just get the Radio Button clicked and saves the ScaleValue selected
+        /// This are the values used for the scales:
+        /// - 2 - Small
+        ///   0 - Medium (Default)
+        ///   2 - Large
+        ///   4 - Extra Large
+        /// </summary>
+        /// <param name="sender"></param>
+        /// <param name="e"></param>
+        private void RadioGeometryScaling_Checked(object sender, RoutedEventArgs e)
+        {         
+            RadioButton selectedScaling = sender as RadioButton;
+            var radioButtons = GeometryScalingRadiosPanel.Children.OfType<RadioButton>();
+
+            int index = 0;
+
+            //We need to loop all the radiobuttons in the GeometryScaling section in order to find the index of the selected one
+            foreach (var radio in radioButtons)
+            {
+                if (radio == selectedScaling)
+                {
+                    if (viewModel != null)
+                        viewModel.SelectedDefaultScaleFactor = GeometryScalingOptions.ConvertUIToScaleFactor(index);
+                    break;
+                }
+                index++;
```

Comment:
hmm I remember radio button supports data binding, this looks more complex than I would expect

Reply:
This code was deleted due that now is using data binding in RadioButtons, check commit: https://github.com/DynamoDS/Dynamo/pull/13656/commits/789f51c4a3c106156940924585e1ca40abd9842b